### ISSUE TITLE
Question about megatron actor

### ISSUE THREAD

**Issue by Cppowboy** [volcengine/verl#227]
In the code verl/workers/actor/megatron_actor.py, could you explain why we append the metric twice?

https://github.com/volcengine/verl/blob/main/verl/workers/actor/megatron_actor.py#L357
```python
for data in dataloader:
            # data = data.batch.to(self.actor_module.device)
            self.actor_optimizer.zero_grad()
            # use use_contiguous_buffers_in_local_ddp and no overlap_dp_param_comm
            for chunk in self.actor_module:
                # if use distributed optimizer, zero grad buffer will be handled by optimizer
                chunk.zero_grad_buffer(zero_buffer=(not self.actor_optimizer_config.use_distributed_optimizer))

            metric_micro_batch = self.forward_backward_batch(data)
            for metric in metric_micro_batch:
                append_to_dict(metrics, metric)  # append the metric from this micro-batch to global metrics.

            update_successful, grad_norm, num_zeros_in_grad = self.actor_optimizer.step(
                self.megatron_config, self.megatron_config.timers)
            if update_successful:
                # allgather already execute in optimizer.step in new megatron
                pass
            else:
                raise NotImplementedError

            for metric in metric_micro_batch:
                append_to_dict(metrics, metric)  # append the metric from this micro-batch to global metrics.
```

**Comment by vermouth1992** [comment]
This should be a typo. Do you mind submit a PR to delete one of them?

**Comment by Cppowboy** [comment]
> This should be a typo. Do you mind submit a PR to delete one of them?

OK, I will submit a PR.
